src/main/java/com/iota/iri/TransactionValidator.java

```diff
@@ -53,15 +52,19 @@ public TransactionValidator(Tangle tangle, TipsViewModel tipsViewModel, Transact
     }
 
     public void init(boolean testnet, int mwm) {
+        setMwm(testnet, mwm);
+
+        newSolidThread = new Thread(spawnSolidTransactionsPropagation(), "Solid TX cascader");
+        newSolidThread.start();
+    }
+
+    void setMwm(boolean testnet, int mwm) {
```

Comment:
just a nit, but `if(!testnet) MIN_WEIGHT_MAGNITUDE = Math.max(MIN_WEIGHT_MAGNITUDE, 13)` would be less magic-number heavy

Reply:
I pulled the code to `setMwm` to make the test work,
The implementation logic is old and not really part of this PR.

But since you commented I will change :-)